graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/DgsRolloutFlags.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs
+
+data class DgsRolloutFlags(
```

Comment:
Instead of a flag to completely enable/disable the feature, could it be opt-in per entity fetcher? E.g. add it as a new optional parameter to `@DgsEntityFetcher`. Something like `@DgsEntityFetcher(name = "Show", enableCustomScalarCoercion=true)`.

That way it will be a bit more easy to gradually adopt this.

Reply:
This is a great idea, and avoids the awkwardness of this class. :)